ew/static/slimeoid.py

```diff
@@ -1266,7 +1261,185 @@ def get_strat_g(combat_data, in_range, first_turn, active):
+        str_movecry="{slimeoid_name} skitters away! ",
+        str_movecry_weak="{slimeoid_name} skitters away quietly. ",
+        str_revive="unused",
+        str_spawn="{slimeoid_name}'s eyes dart frantically around the room. It sees you and carefully approaches, before the Waffle House worker comes back! She smacks the shit out of {slimeoid_name} with a spatula, before running to the back to grab some fryer grease. It'd probably be best if you got out of here.",
+        str_dissolve="unused",
+        get_strat=get_strat_e,
+        str_abuse="{slimeoid_name} takes the hit, crying."
+    ),
+    EwBrain(  # brain 13
+        id_brain="m",
+        alias=[
+            "typem",
+            "type m"
+        ],
+        str_create="You move the planchette to spell \"BRAIN\", then move the planchette to \"F\". \nA cluster of antineurons coagulates within your newly-forming Negaslimeoid. The proto-Negaslimeoid looks up towards you from the floor, watching your every move.",
+        str_brain="It's silently observing all those around it.",
+        str_observe="You go to look at {slimeoid_name}, but it was already staring at you. Unblinking.",
+        str_pet="{slimeoid_name} looks at your hand. You're a bit too unsettled to actually pet it.",
+        str_walk="{slimeoid_name} gets put on a leash, and moves along with you, staring at its surroundings.",
+        str_feed="{slimeoid_name} only looks at the {food_name}. You mimic eating the food, and eventually {slimeoid_name} begins to slowly peck at it.",
+        str_kill="unused",
+        str_death="unused",
+        str_victory="{slimeoid_name} looks at the remains of its opponent, unmoving.",
+        str_battlecry="{slimeoid_name} lets out a quiet coo. ",
+        str_battlecry_weak="{slimeoid_name} sheds single tear. ",
+        str_movecry="{slimeoid_name} tactically stays within range!. ",
+        str_movecry_weak="{slimeoid_name} tries to tactically stay within range. ",
+        str_revive="unused",
+        str_spawn="{slimeoid_name} carefully picks itself up, then looks at you. It looks out the blackened windows of Waffle House, chittering at its reflection. It doesn't seem to like this place.",
+        str_dissolve="unused",
+        get_strat=get_strat_f,
+        str_abuse="{slimeoid_name} saw your abuse coming, but isn't fast enough to get beat up."
+    ),
+    EwBrain(  # brain 14
+        id_brain="n",
+        alias=[
+            "typen",
+            "type n"
+        ],
+        str_create="You move the planchette to spell \"BRAIN\", then move the planchette to \"G\". \nA cluster of antineurons coagulates within your newly-forming Negaslimeoid. The proto-Negaslimeoid begins muttering incomprehensibly.",
+        str_brain="It's muttering  ̷̢̧̛̞̮͕̫͂̀͐̅̀̏̓̏̽̅͋̈́̌̃̽͘͘̚͝͠͝▐̷̨̢̲͙̦̤̮̞͈̠̬͖̀̈̂̅͗̑̓͝▄̸̡̧̧̛̱͓̜͍͎̹̟̣̰̞̤̹̝̹̼̘̺̳̈́̔́͑͐̑͋͛̆̄̅̍͘͝█̷̧̨̛͉͚̟͍̙͓̻͖̝̗̅́͊͜ ̶̡̗̺̦͉̲̝̼͗͗█̶̠̗͔͙͖̺̣̑̃͌̆̑̒͂͗̾̒̋̔͛̚̕ ̴̢̨̰̝͇̘͒́̔̀̉͆̇͝▮̸̧̛̛̜̱͇̰͙͉̩̭̘̠̞̹̜̳̤͖̯͚̳̇̃̄̇̏̐̈́͒͂͋̑͊̎͆͠⯊̶̡̛̘͕͈̀͆̅̍͂͌͠  to itself.",
```

Comment:
this'll be sick, as long as it works. do you know if it displays like this in discord?